src/app/space/create/deployments/components/deployment-card/deployment-card.component.ts

```diff
@@ -24,21 +26,37 @@ export class DeploymentCardComponent implements OnDestroy, OnInit {
   @Input() applicationId: string;
   @Input() environment: Environment;
 
-  public data: any = {
+  public cpuData: any = {
     dataAvailable: true,
     total: 100,
-    xData: ['foo', 1, 2, 3, 4],
-    yData: ['used', 10, 20, 30, 40]
+    xData: ['time', 0],
+    yData: ['used', 1]
   };
 
-  public config: any = {
+  public memData: any = {
+    dataAvailable: true,
+    total: 100,
+    xData: ['time', 0],
+    yData: ['used', 1]
+  };
+
+  public cpuConfig: any = {
+    // Seperate chart IDs must be unique, otherwise only one will appear
+    chartId: 'cpu-chart-' + DeploymentCardComponent.chartIdNum++ + '-'
+  };
+
+  public memConfig: any = {
     // Seperate chart IDs must be unique, otherwise only one will appear
-    chartId: 'chart-' + DeploymentCardComponent.chartIdNum++ + '-'
+    chartId: 'mem-chart-' + DeploymentCardComponent.chartIdNum++ + '-'
```

Comment:
Please use this function to generate the unique ID:
https://lodash.com/docs/4.17.4#uniqueId

This matches with Jie's donut component and prevents us from having to store some static (basically global) state for counting how many charts there are.